src/AWS.ts

```diff
@@ -10,6 +10,11 @@ import AWS from 'aws-sdk';
 const AWSWithXray = AWSXRay.captureAWS(AWS);
 
 const { IS_OFFLINE } = process.env;
+
+AWS.config.update({
+    customUserAgent: process.env.CUSTOM_USER_AGENT,
```

Comment:
Does it fail gracefully when `CUSTOM_USER_AGENT` is not set?

Reply:
Yep, `customUserAgent` accepts a `string` or `undefined`. Then the sdk appends that string to the userAgent field.